messageBundles/gpii-app-messageBundle_bg.json

```diff
@@ -27,7 +27,12 @@
 
     "gpii_psp_errorDialog_titlebarAppName": "GPII Грешка",
 
-    "gpii_psp_aboutDialog_titlebarAppName": "Относно GPII Auto-personalization",
+    "gpii_psp_aboutDialog_titlebarAppName": "Относно GPII Авто-персонализиране",
+    "gpii_psp_aboutDialog_title": "GPII Авто-персонализиране",
+    "gpii_psp_aboutDialog_titlebarAppName": "Относно GPII Авто-персонализиране",
```

Comment:
Remove this as it is the second occurrence of the `titlebarAppName`. The same applies to the `..._en.json`.